test_runner/src/main/kotlin/ftl/util/Utils.kt

```diff
@@ -117,7 +117,7 @@ fun copyBinaryResource(name: String) {
 
     // "binaries/" folder prefix is required for Linux to find the resource.
     Files.copy(
-        getResource("binaries/$name"),
+        getResource(name),
```

Comment:
getResource("binaries/$name"),
```suggestion
        getResource("binaries/$name"),
```